### PR TITLE
translate: basic/simulate.md

### PR DESCRIPTION
**Pull request by CamberLoid**
Please **Reviewers** take care of the following aspects:

1. Are there any typos?
2. Please read the content of the Pull Request as if you were a beginner and talk about your thoughts after reading the text no matter whether you are familiar with the content or not.
3. Please evaluate the content of the Pull Request if you are familiar with related concepts.
4. Please **try your best** to keep an eye on it until it is merged or closed.

### CODE REVIEW

File: docs/basic/simulate.md [hunk 1]
```diff
@@ -1,17 +1,55 @@
...
+Last translate with upstream: [58b9fe6](https://github.com/OI-wiki/OI-wiki/commit/58b9fe67d667d265261d733b70b90e6cf0eaefdd) on Feb 8, 2021.
 
-Of course, simulations are not always easy to write. See classic problems like [World of Warcraft] (http://bailian.openjudge.cn/practice/3750/) and [pig kingdom kill] (https://loj.ac/problem /2885) for reference.
+This article will briefly introduce implementation method.
 
-Simulation problems usually have large code size, many operations, and complicated logics. And because of its large amount of code, the debugging process is really difficult. Therefore, it would be quite time-consuming if you made mistakes during contests.
+## Introduction
 
-Therefore, for simulation problems, the following suggestions below may help you improve the speed:
+Implementation, or so called simulation in China, as the name implies, is the method that computers use to simulate the operations required in the problem.
```
Comment (ksyx) [inline docs/basic/simulate.md:7]:
```suggestion
Implementation, also known as Simulation, as the name implies, is the method that use computers to simulate the operations required in the problem.
```
只要有人用就可以 aka（

File: docs/basic/simulate.md [hunk 2]
```diff
@@ -1,17 +1,55 @@
...
+- For some concepts that may be reused, they can be transformed in a unified way so they are easier to handle: for example, if a problem gives you "YY-MM-DD hour:minute", processing it in a function into seconds format will help reduce confusion;
+- When debugging, debug in blocks. The advantage of modularity is that it is convenient to adjust a certain part separately;
+- When writing the code, think clearly. Do not think intuitively, instead, follow the steps you have written on the scratch paper.
 
 In fact, the above steps are also very helpful in solving other types of problems.
+
+## Example
+
+???+note " [Climbing Worm - HDU](http://acm.hdu.edu.cn/showproblem.php?pid=1049"
+    An inch worm is at the bottom of a well n inches deep. It has enough energy to climb u inches every minute, but then has to rest a minute before climbing again. During the rest, it slips down d inches. The process of climbing and resting then repeats. How long before the worm climbs out of the well? We'll always count a portion of a minute as a whole minute and if the worm just reaches the top of the well at the end of its climbing, we'll assume the worm makes it out.
```
Comment (ksyx) [inline docs/basic/simulate.md:26]: 输入的变量加一下公式

File: docs/basic/simulate.md [hunk 3]
```diff
@@ -1,17 +1,55 @@
...
+      }
+      return 0;
+    }
+    ```
+    
+## Exercises
+
+- (Chinese) [NOIP2014 Rock, Paper, Scissors, Lizard, Spock](https://uoj.ac/problem/15)
+- (Chinese) [3750 World of Warcraft - OpenJudge](http://bailian.openjudge.cn/practice/3750/)
+- (Chinese) [SDOI2010: Pig Kingdoms slaughter](https://loj.ac/p/2885)
```
Comment (ksyx) [inline docs/basic/simulate.md:55]: Chinese 后置吧

**Review by ksyx — COMMENTED** [review]
thx

File: docs/basic/simulate.md (hunk 1 above)
Comment (CamberLoid) [reply docs/basic/simulate.md:7]: pwp

**Comment by CamberLoid** [comment]
https://github.com/OI-wiki/i18n/pull/126#discussion_r689079451
https://github.com/OI-wiki/i18n/pull/126#discussion_r689079493
https://github.com/OI-wiki/i18n/pull/126#discussion_r689079146

fixed in 1c41e43c

File: docs/basic/simulate.md [hunk 4]
```diff
@@ -1,17 +1,55 @@
...
-3. For some concepts that may be reused, they can be transformed in a unified way so they are easier to handle: for example, if a problem gives you "YY-MM-DD hour:minute", processing it in a function into seconds format will help reduce confusion;
-4. When debugging, debug in blocks. The advantage of modularity is that it is convenient to adjust a certain part separately;
-5. When writing the code, think clearly. Do not think intuitively, instead, follow the steps you have written on the scratch paper.
+## Tricks for writing 
 
+When writing solution for implementation problems, the following suggestions below may help you improve the speed:
+
+- Before writing the code, write down the process of your implementation on scratch paper as complete as possible;
+- In the code, try to modularize each part as a function, structure, or class;
+- For some concepts that may be reused, they can be transformed in a unified way so they are easier to handle: for example, if a problem gives you "YY-MM-DD hour:minute", processing it in a function into seconds format will help reduce confusion;
```
Comment (ksyx) [inline docs/basic/simulate.md:17]:
```suggestion
- For some concepts that may be reused, they can be transformed in a unified way so they are easier to handle: for example, if a problem gives you "YY-MM-DD hour:minute", use a function to turn it into seconds format will help reduce confusion;
```

File: docs/basic/simulate.md [hunk 5]
```diff
@@ -1,17 +1,55 @@
...
 
 In fact, the above steps are also very helpful in solving other types of problems.
+
+## Example
+
+???+note " [Climbing Worm - HDU](http://acm.hdu.edu.cn/showproblem.php?pid=1049"
+    An inch worm is at the bottom of a well $n$ inches deep. It has enough energy to climb $u$ inches every minute, but then has to rest a minute before climbing again. During the rest, it slips down $d$ inches. The process of climbing and resting then repeats. How long before the worm climbs out of the well? We'll always count a portion of a minute as a whole minute and if the worm just reaches the top of the well at the end of its climbing, we'll assume the worm makes it out.
+
+??? note "Solution"
+    Directly implementing the process of worm's climbing will do the trick. Use a loop statement to repeat the process of climbing, and jump out when the climbed length greater than or equal to the depth of well. Note that both ascending and descending have time cost.
```
Comment (ksyx) [inline docs/basic/simulate.md:29]: > do the trick

正式点更好？

**Pull request merged**